### PR TITLE
[zh] synchronize translate validate-dual-stack.md

### PR DESCRIPTION
**Pull request by shuaijinchao**
/language zh

### CODE REVIEW

File: content/zh/docs/tasks/network/validate-dual-stack.md [hunk 1]
```diff
@@ -31,6 +31,15 @@ This document shares how to validate IPv4/IPv6 dual-stack enabled Kubernetes clu
   （如 Calico，Cilium 或 Kubenet）。 
 * [启用双协议栈](/zh/docs/concepts/services-networking/dual-stack/) 集群
 
+{{< version-check >}}
+
+<!--
+While you can validate with an earlier version, the feature is only GA and officially supported since v1.23.
+-->
+{{< note >}}
+虽然你可以使用较早的版本进行验证，但该功能只是 GA 并从 v1.23 版本开始得到正式支持。
```
Comment (tengqm) [inline content/zh/docs/tasks/network/validate-dual-stack.md:40]:
```suggestion
虽然你可以使用较早的版本进行验证，但该功能是从 v1.23 版本进入 GA 状态并正式支持的。
```

Comment (shuaijinchao) [reply content/zh/docs/tasks/network/validate-dual-stack.md]: done

**Pull request merged**